crates/js-component-bindgen/src/intrinsics.rs

```diff
@@ -96,6 +97,12 @@ pub fn render_intrinsics(
                 const hasOwnProperty = Object.prototype.hasOwnProperty;
             "),
 
+            Intrinsic::ToResultString => output.push_str("
+                function toResultString(obj) {
+                    return JSON.stringify(obj, (_, v) => typeof v === 'bigint' ? v.toString() : v);
```

Comment:
Just wondering if this might be quite noisy in the case of big return objects. Perhaps we can make the tracing a config object, and make this an option, `{ results: true }`, say?

Reply:
i would rather us limit the string size and/or object depth (like [here](https://github.com/formfunction-hq/formfunction-monorepo/blob/6d049d5b1ff852e6d9f0126b80924e989e1f257e/packages/shared/src/utils/jsonStringify.ts#L9)) with some sensible defaults.

Reply:
I think blanket turning off would be useful but we can land separately as well. For example - currently `new Uint8Array([1,2,3])` will output its bytes in this position, which is definitely not what we want.

Reply:
Can we just disable array buffers perhaps for now?

Reply:
instances of `Uint8Array` have been disabled. your example will now print `... result="[Uint8Array (3)]"`